.github/workflows/nightly.yml

```diff
@@ -292,6 +305,45 @@ jobs:
           docker rmi test-image
           rm -rf app/tomee
 
+      - name: Run Archetype for EE 8 Core Profile, SE 8, TomEE
```

Comment:
You were right the first time. Since TomEE does not support EE 10 yet, we cannot meaningfully do this test. Let’s remove it. Thanks!